pkg/util/workloads_utils.go

```diff
@@ -212,6 +226,95 @@ func GenRandomStr(length int) string {
 	return rand.SafeEncodeString(randStr)
 }
 
+func PatchUpdateStrategy(c client.Client, object client.Object, updateStrategy map[string]interface{}) error {
+	patch := map[string]interface{}{
+		"spec": map[string]interface{}{
+			"updateStrategy": updateStrategy,
```

Comment:
This should not be a common logic in util, for workloads like Deployment may have field named `strategy` instead of `updateStrategy`.